stdlib/universe/moving_average.go

```diff
@@ -72,9 +70,8 @@ func (s *MovingAverageOpSpec) Kind() flux.OperationKind {
 
 type MovingAverageProcedureSpec struct {
 	plan.DefaultCost
-	N          int64    `json:"n"`
-	Columns    []string `json:"columns"`
-	TimeColumn string   `json:"timeColumn"`
+	N       int64    `json:"n"`
+	Columns []string `json:"columns"`
```

Comment:
Did we decide for columns or column?

Reply:
I think this should contrast with the `timed` version.
Why should that have `column` and this one `column`?
We will bring this into discussion with the team.

Reply:
I agree, why columns instead of column?